lua/entities/gmod_tardis/modules/sh_teleport.lua

```diff
@@ -184,6 +205,11 @@ if SERVER then
 				end
 			end
 		end
+		if self:GetData("demat-fast",false) then
+			timer.Simple(2,function()
```

Comment:
what's the reason behind this timer (personally not a big fan of using timer.Simple for stuff it doesn't pause with the game and things like that